application/hubspot/v0/contact.go

```diff
@@ -0,0 +1,32 @@
+package hubspot
```

Comment:
Some senior gopher said like this.
[Reference](https://stackoverflow.com/a/26229922)

So, let's arrange the knowledge into the same files and avoid to make a file contains only struct.

Reply:
Yap, just like mentioned above, I will change it, and will ping you once again once I finish. Thanks for the advice!